src/app/components/tabBar/index.tsx

```diff
@@ -60,22 +60,26 @@ function BottomTabBar({ tab }: Props) {
 
   const showBottomBar = !isInOptions();
 
-  return showBottomBar ? (
+  return (
     <RowContainer>
-      <Button onClick={handleDashboardButtonClick}>
-        <img src={tab === 'dashboard' ? WalletTab : UnselectedWalletTab} alt="dashboard" />
-      </Button>
-      <Button onClick={handleNftButtonClick}>
-        <img src={tab === 'nft' ? NftTab : UnselectedNftTab} alt="nft" />
-      </Button>
-      <Button onClick={handleStackingButtonClick}>
-        <img src={tab === 'stacking' ? StackingTab : UnselectedStackingTab} alt="stacking" />
-      </Button>
-      <Button onClick={handleSettingButtonClick}>
-        <img src={tab === 'settings' ? SettingsTab : UnselectedSettingsTab} alt="settings" />
-      </Button>
+      {showBottomBar && (
```

Comment:
I think this changes the functionality a bit. before, if showBottomBar was falsey, it would return null. Now, it will always render the RowContainer which has a minHeight of 64px, so it would render an empty div. Is that the expected behaviour?

Reply:
Yeah sorry didn't document this change, but it's the expected new behaviour in revamped design

Reply:
![Uploading image.png…]()